### PR TITLE
fix(pagination): Remove space between pagination numbers in v4

### PR DESCRIPTION
**Pull request by hannahiss**
Also remove margin at start

Backport of #1120 to v4

### CODE REVIEW

File: scss/_pagination.scss [hunk 1]
```diff
@@ -209,6 +208,17 @@
     }
   }
   // End mod
+
+
+  // Boosted mod
```
Comment (louismaximepiton) [inline scss/_pagination.scss:213]: Could be packed with the previous one IMO

File: scss/_variables.scss [hunk 2]
```diff
@@ -932,6 +932,7 @@ $dropdown-header-padding:           $dropdown-padding-y !default; // Boosted mod
 // Pagination
 
 $pagination-margin-y:               $spacer !default;
+$pagination-margin-x:               .5rem !default; // Boosted mod
```
Comment (louismaximepiton) [inline scss/_variables.scss:935]: Should be `.625rem`

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Focus state on the active one looks kind of weird to me

File: scss/_variables.scss (hunk 2 above)
Comment (hannahiss) [reply scss/_variables.scss:935]: ah oui, j'ai gardé la même valeur sans vérifier sur le deisgn...

**Review by louismaximepiton — APPROVED** [review]
The selectors aren't the same in v4 and v5, does that impact anything ? 
However, LGTM

**Review by julien-deramond — APPROVED** [review]
LGTM! FYI made some minor changes in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1269/commits/6d12751b1f09ca9ffbc03a9b3b19e020b13e3fe2.

**Comment by julien-deramond** [comment]
> The selectors aren't the same in v4 and v5, does that impact anything ? However, LGTM

Just to answer to your question @louismaximepiton. The structure is not exactly the same so no problem to have different selectors in v4 and v5 in this case.

**Pull request merged**